python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -446,10 +448,13 @@ def _parse_message_data(message_data: Optional[Mapping[str, Any]]) -> Iterator[T
             if isinstance(content, str):
                 yield MESSAGE_CONTENT, content
             elif isinstance(content, list):
-                for i, obj in enumerate(content):
-                    assert hasattr(obj, "get"), f"expected Mapping, found {type(obj)}"
+                if isinstance(obj, str):
+                    yield f"{MESSAGE_CONTENTS}.0", obj
```

Comment:
this one is a value returned by the LLM.
so sometimes the LLM returns a `list[str]`. while usually it returns just a single `str`?
fwiw I'm using `langchain_google_vertexai.VertexAI`